mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -21,6 +21,73 @@ message CreateRegisteredModel {
   }
 }
 
+message DeleteRegisteredModel {
+  // Required. Full name of the model.
+  optional string full_name = 1 [(validate_required) = true];
+  // Optional. Force deletion even if the registered model is not empty. Default is false.
+  optional bool force = 2;
+
+  message Response {}
+}
+
+message GetRegisteredModel {
+
+  // Required. Full name of the Model.
+  optional string full_name = 1 [(validate_required) = true];
+
+  message Response {
+    optional RegisteredModelInfo registered_model_info = 1 [(json_inline) = true];
+  }
+}
+
+message CreateModelVersion {
+  optional ModelVersionInfo model_version_info = 1 [(json_inline) = true];
+
+  message Response {
+    optional ModelVersionInfo model_version_info = 1 [(json_inline) = true];
+  }
+}
+
+message UpdateRegisteredModel {
+
+  // Required. Full name of the Model.
+  optional string full_name = 1;
+
+  // Optional. New name of registered model (replaces registered_model_info.name for specifying new name).
+  optional string new_name = 3;
+
+  // Optional. Partial RegisteredModelInfo with fields to change.
+  optional RegisteredModelInfo registered_model_info = 2 [(json_inline) = true];
```

Comment:
The only thing we can update is the comment.

See: https://docs.databricks.com/api/workspace/registeredmodels/update

Because we arent using the json_inline annotation here, we just need to define the comment field.